neard/Cargo.toml

```diff
@@ -3,6 +3,7 @@ name = "neard"
 version = "1.0.0"
 authors = ["Near Inc <hello@nearprotocol.com>"]
 edition = "2018"
+default-run = "neard"
```

Comment:
what's this?

Reply:
This is `cargo run -p neard` works now. And before it would require to specify also `--bin neard`